### DISCUSSION TITLE
404 of https://github.com/argoproj/argo-rollouts/releases/latest/download/notifications-install.yaml

### DISCUSSION THREAD

**Discussion by 13567436138** [argoproj/argo-rollouts#1477] [Q&A] [answered]
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

resources:
- https://github.com/argoproj/argo-rollouts/releases/latest/download/install.yaml
- https://github.com/argoproj/argo-rollouts/releases/latest/download/notifications-install.yaml

**Comment by kostis-codefresh** [comment, marked as answer]
This has been fixed.
